addons/account/views/product_views.xml

```diff
@@ -16,14 +16,15 @@
          <field name="model">product.product</field>
          <field name="inherit_id" ref="product.product_product_view_form_normalized"/>
          <field name="arch" type="xml">
-             <xpath expr="//div[@name='sales_price']" position="inside">
-                 <field name="tax_string"/>
-             </xpath>
-             <xpath expr="//div[@name='sales_price']" position="after">
-                 <field name="taxes_id" widget="many2many_tags"
-                        context="{'default_type_tax_use': 'sale', 'search_default_sale': 1}"
+            <field name="list_price" position="after">
+                <label for="taxes_id"/>
+                <div name="tax_info" class="d-flex">
```

Comment:
no need to add d-flex I think :thinking:

Reply:
No no that is just added to show the fields in a single line to improve ui like this
![image](https://github.com/user-attachments/assets/3959c5fc-7e6d-4d8c-b90d-e759723fbbe4)
Otherwise they looks like this
![image](https://github.com/user-attachments/assets/f8cbd1e0-a6d9-4155-acbc-74ce0be7bd71)

Reply:
Got it but in odoo we use `o_row` class :thinking: 
So it'll also add some other property with d-flex like add little space b/w input and string 
![image](https://github.com/user-attachments/assets/399454db-1649-4f20-a1c4-f85c28700c39)

and some other things